system-int-test/test/cmr/system_int_test/search/service/collection_service_search_test.clj

```diff
@@ -499,3 +499,47 @@
 
         "multiple search search"
         [coll1 coll2 coll3 coll4] ["Harmony" "OPeNDAP"]))))
+
+(deftest collection-services-search-has-combine-test
+  (let [token (e/login (s/context) "user1")
+        coll1 (d/ingest "PROV1" (dc/collection {:entry-title "ET1"
+                                                :short-name "S1"
+                                                :version-id "V1"}))
+        coll2 (d/ingest "PROV1" (dc/collection {:entry-title "ET2"
+                                                :short-name "S2"
+                                                :version-id "V2"}))
+        coll3 (d/ingest "PROV1" (dc/collection {:entry-title "ET3"
+                                                :short-name "S3"
+                                                :version-id "V3"}))
+        ;; create services
+        {serv1-concept-id :concept-id}
+        (service-util/ingest-service-with-attrs
+         {:native-id "serv1"
+          :Type "Harmony"
+          :Name "service1"})
+
+        {serv2-concept-id :concept-id}
+        (service-util/ingest-service-with-attrs
+          {:native-id "serv2"
+           :Name "service2"
+           :Type "Harmony"
+           :ServiceOptions {:Aggregation {:Concatenate {:ConcatenateDefault true}}}})
+
+        {serv3-concept-id :concept-id}
+        (service-util/ingest-service-with-attrs
+         {:native-id "serv3"
+          :Name "service3"
+          :Type "Harmony"
+          :ServiceOptions {:Aggregation {:Concatenate {:ConcatenateDefault false}}}})]
+
+    (index/wait-until-indexed)
+    (au/associate-by-concept-ids token serv1-concept-id [{:concept-id (:concept-id coll1)}])
+    (au/associate-by-concept-ids token serv2-concept-id [{:concept-id (:concept-id coll2)}])
+    (au/associate-by-concept-ids token serv3-concept-id [{:concept-id (:concept-id coll3)}])
+    (index/wait-until-indexed)
+    (testing "when Aggregation is not present"
+      (service-util/assert-collection-search-result coll1 {:has-combine false :service-features {:harmony {:has-combine false}}} [serv1-concept-id]))
+    (testing "when concatenateDefault is true"
+      (service-util/assert-collection-search-result coll2 {:has-combine true :service-features {:harmony {:has-combine true}}} [serv2-concept-id]))
+    (testing "when concatenateDefault is false"
```

Comment:
Thinking about this more closely in this case, even if the value for `ConcatenateDefault` is `false` don't we still have the option to do `combines` because the `serviceOption` obj of `:Concatenate` does exist.
Should `has_combine` be true in that case?

Reply:
So on EDSC I think we would still want to show the Badge even if the `concatenateDefault` is false

Reply:
Yes, the existence of the UMM-S field in the metadata means that the data set can be concatenated.  It is irrelevant if the value is true or false.  Also can we change the name from has-combined to has-concatenate as concatenate has a specific meaning to the subsetting people and it aligns better to the UMM-S field that holds the data.

Reply:
The ticket, which you wrote, says "On the collection's JSON response, we should add a field called has_combine which is a boolean that is determined whether or not the collection is associated to a UMM-S record that contains an InstanceInformation field where the sub field is true such as this:"
```   
 "aggregation": {"concatenate": {"concatenateDefault": true} 
```

Until the ticket changes the implementation will remain this way.

Reply:
Ah I didn't see erich's update, will adjust then.

Reply:
I understand what happen.  Leaving the name is good.